src/webkit/webkit.c

```diff
@@ -563,31 +523,36 @@ liferea_webkit_default_settings (WebKitSettings *settings)
 	);
 }
 
+static LifereaWebKit *liferea_webkit = NULL;
+
 /**
- * Initializes WebKit
- *
- * Initializes the WebKit HTML rendering engine. Creates a WebKitWebView.
+ * Create new WebkitWebView object and connect signals to a LifereaHtmlview
  */
-static GtkWidget *
-liferea_webkit_new (LifereaHtmlView *htmlview)
+GtkWidget *
+liferea_webkit_new (LifereaBrowser *htmlview)
 {
 	WebKitWebView 	*view;
 	WebKitSettings	*settings;
 
+	if (!liferea_webkit) {
+		liferea_webkit = g_object_new (LIFEREA_TYPE_WEBKIT, NULL);
+		liferea_webkit_init (liferea_webkit);
```

Comment:
I think this is superfluous; `g_object_new` calls init. I am getting a critical warning that the liferea scheme is being registered twice unless I remove this line.

Reply:
You are right, fixed with 20340425